pyproject.toml

```diff
@@ -28,6 +28,7 @@ version = "0.0.0"
 aiohttp = ">=3.0.0"
 python = "^3.11"
 yarl = ">=1.6.0"
+aiozoneinfo = "^0.2.1"
```

Comment:
Instead of adding a dependency, we can also move this one out of the function scope and put it e.g., into a constant. The timezone isn't dynamic.

../Frenck

Reply:
That still makes it a class member.

Instead, it can be just defined as a constant at the top of the file and use the constant later on.

Reply:
Ah oh, sorry i misunderstood, i don't write python usually, so didn't know the Usual style rules,
I've made it into a const now